webapp/views/launch/script.js

```diff
@@ -583,8 +581,21 @@ function saveDrawing(res) {
   return true;
 }
 
-function getOutput(res, message) {
-  sendMessage('/output', { msg: message }, { message_id: res.message_id }, handleResponse);
+function getOutput() {
+  sendMessage('/output', {}, {}, function(res){
+    $('#results-select').html(function () {
+      const html = "<option selected value=''> - </option>" + res.message.list.reduce((str, file) => str + `<option value="${file}">${file}</option>`, '');
+      return html
+    })
```

Comment:
instead of a function, you can simply pass a string to html()

Reply:
fixed here 943d4714b310778e04708c5a776551752e06b442